source/basic.tex

```diff
@@ -2520,7 +2520,7 @@
 s)~(\ref{multibyte.strings}) and \tcode{argv[0]} shall be the pointer to
 the initial character of a \ntmbs that represents the name used to
 invoke the program or \tcode{""}. The value of \tcode{argc} shall be
-non-negative. The value of \tcode{argv[argc]} shall be 0. \begin{note} It
+non-negative. The value of \tcode{argv[argc]} shall be the null pointer value. \begin{note} It
```

Comment:
Also in the other occurrences below. We can have "the null pointer value of some type", but "the null pointer value" without a type doesn't make sense, whereas "a null pointer value" seems to work.